helper/flags/flags.go

```diff
@@ -24,17 +24,21 @@ func (i *ArrayFlags) Set(value string) error {
 	return nil
 }
 
-type BootnodeFlags []string
+type BootnodeFlags struct {
+	IsSet bool
```

Comment:
```suggestion
	AreSet bool
```

Since this bool is in regards to the bootnode flags (multiple) it should reference them in plural form

Reply:
Renamed it , Commit [link](https://github.com/0xPolygon/polygon-sdk/pull/260/commits/abff6809bcbc43529eff0a1b051caa06584c7c7e)